languages/en.ini

```diff
@@ -246,6 +246,7 @@ collection_disambiguation = "Found Multiple Matching Collections"
 collection_empty = "No items to display."
 collection_view_record = "View Record"
 Collections = "Collections"
+combined_more_advanced_separator = "or use"
```

Comment:
I strongly recommend using a complete translation instead of stitching the result from multiple translations. Something like this would work and be much easier to translate:
`combined_more_advanced_link_html = "or use <a href="%%url%%">Advanced Search</a>"`

See e.g. `alphabrowselink_html` for existing usage.

Reply:
I think the idea here is that there could be just an advanced link or a more link or both... and there's a separator in between the two links only in the case where both are displayed. I think trying to do this as a single language string could get awkward, but I also agree that stringing strings together is not ideal either. I like the fact that the string is named as a separator to make its purpose clear... but then based on the initial screen shot above, it doesn't really look like a separator since it's just part of a continuous sentence.

I realize my comments are not necessarily pointing toward any particular solution... but just trying to clarify the problems. :-)

Reply:
In that case we may need two different translation strings. Perhaps not so elegant, but easier to manage! :)

Reply:
@demiankatz is correct that I did this because each of the two links may be there on its own; the separator is only used when both are.  I could create translations that accounted for all three possibilities if you all prefer that.  This seemed (slightly) simpler.

I could also simplify the whole thing by making the advanced link a sub-option of the more-link, i.e. there is no possibility of advanced link alone.  But that seemed wrong.

I can do the two separate translation strings if folks prefer that, but that's what's effectively in combined.ini since it's also an enable/disable toggle.

Reply:
Could we do without the "or use" part? That would simplify things, and since there's the icon and line break between the two links, they're not so naturally parts of a sentence. The problem is that Starting with "or use" may cause the rest to need a different inflexion or something (typical with Finnish), which makes it difficult to formulate in a way that fits both cases.

Reply:
@EreMaijala Personally I am fine with that.  For my local theme I would likely add the "or use" part via css i.e. 

`p.advanced_link:before { content: 'or use'; }`

@demiankatz Do you agree?

Reply:
Done.

![image](https://github.com/vufind-org/vufind/assets/31278545/82539e6a-e113-4389-94e9-5c9229ed1ff8)